rero/sonar
# organisation: improve organisation editor

* Moves organisation code from "pid" to "code" field.
* Configures JSON schema for improving organisation editor.
* Adds a method to retrieve organisation by its code in OrganisationRecord.
* Renames tests file and folder containing "institution".
* Removes useless registered_organisations property from Overdo class.
* Closes rero/sonar-ui#64.

Co-Authored-by: Sébastien Délèze <sebastien.deleze@rero.ch>

**How to test**
1. Login as an administrator.
2. Go to Admin -> Records -> Organisations.
3. Click Add button.
4. Fill the form and check field requirements and validations.
5. Save organisation.
6. Edit organisation and change some fields.
7. Save organisation.
8. Delete organisation.
